docs/platform/api/linode-cli/index.md

```diff
@@ -197,6 +197,10 @@ Tasks related to Linode instances are performed with `linode-cli linodes [ACTION
 1.  Rebuild a Linode:
 
         linode-cli linodes rebuild $linode_id --image linode/debian9 --root_pass
+        
+1. Rebuild a Linode, adding a populated authorized_keys file:
+
+        linode-cli linodes rebuild $linode_id --image linode/debian9 --root_pass --authorized_keys "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIN4f0uMlX/9kH/6yRuHQw8m8qd5niKV9fTDQh90N0wMe"
```

Comment:
you could also use `--authorized_keys "$(cat ~/.ssh/id_rsa.pub)"` as a slightly more generic example